### ISSUE TITLE
Docs 

### ISSUE THREAD

**Issue by EthnTuttle** [ngutech21/moksha#219]
Followed the docs and found I had to `sudo apt install protobuf-compiler` to get `just run-mint` to work.
fwiw, this is on Ubuntu 22.04

**Comment by EthnTuttle** [comment]
for dev, the example config should have `MINT_API_PREFIX=/api` commented out or `just run-cli info` fails with 
```
ethantuttle@panda:~/code/moksha$ just run-cli info
RUST_BACKTRACE=1 cargo run --bin moksha-cli -- -m http://127.0.0.1:3338 -d ./data/wallet  info 
    Finished dev [unoptimized + debuginfo] target(s) in 0.15s
     Running `target/debug/moksha-cli -m 'http://127.0.0.1:3338' -d ./data/wallet info`
Error: Mint does not support /v1 api
error: Recipe `run-cli` failed on line 61 with exit code 1
```

**Comment by ngutech21** [comment]
Thanks for the feedback. Fixed it 487ee55cf99c30d3f43d85f392e198fba714c2c5
